storm-client/test/jvm/org/apache/storm/tuple/ValuesTest.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.storm.tuple;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ValuesTest {
+
+    @Test
+    public void testNoArgsToValues() {
+        Values vals = new Values();
+        Assert.assertTrue("Failed to add null to Values", vals.size() == 0);
+    }
+
+    @Test
+    public void testNullArgsToValues() {
+        Values vals = new Values(null);
+        Assert.assertTrue("Failed to add null to Values", vals.size() == 1);
```

Comment:
I'm sorry to comment this too late, but let's make sure checking elements in values. Same applies to other tests.

Reply:
@HeartSaVioR, Make changes to unit tests to explicitly check elements in values.